docs/examples/infer-orientation-fit-pvwatts.py

```diff
@@ -0,0 +1,88 @@
+"""
+Infer System Orientation (Azimuth & Tilt) using PVWatts
+=======================================================
+
+Infer the azimuth and tilt of a system using PVWatts data
+"""
+
+# %%
+# Identifing and/or validating  the azimuth and tilt information for a
+# system is important, as these values must be correct for future degradation
+# and system yield analysis. This example shows how to use
+# :py:func:`pvanalytics.system.infer_orientation_daily_peak` to estimate
```

Comment:
Copy/paste error

Reply:
Resolved in https://github.com/pvlib/pvanalytics/pull/146/commits/7ee573577bc704d95d49e2b0b8a02d5688184c47